code/game/objects/items/weapons/storage/bags.dm

```diff
@@ -418,6 +418,7 @@
 	..()
 
 /obj/item/storage/bag/tray/cyborg
+	cant_hold = list(/obj/item/disk/nuclear) // Prevents some cheesing
```

Comment:
I feel its worth defining this on the `/tray` level to account for normal ones. A lot of storage types ban the NAD outright (plastic bags & trashbags to name a few)